src/Components/DetailPage/DetailProps.js

```diff
@@ -0,0 +1,208 @@
+import React from "react";
+import DetailSupport from "react";
+import styled from "styled-components";
+
+const DetailProps = (props) => {
+  const propsList = [
+    props.duty,
+    props.qualification,
+    props.preference,
+    props.benefit,
+  ];
```

Comment:
```suggestion
const DetailProps = ({ duty, qualification, preference, benefit }) => {
  const propsList = [duty, qualification, preference, benefit];
```
이렇게 하면 전반적으로 간단해지겠네요~